terragraph_planner/common/rf/link_budget_calculator.py

```diff
@@ -278,7 +278,8 @@ def get_fspl_based_net_gain(
     rx_radio_pattern_data: Optional[Union[AntennaPatternData, ScanPatternData]],
     tx_deviation: float,
     rx_deviation: float,
-    el_deviation: float,
+    tx_el_deviation: float,
```

Comment:
@pohanhf do you agree with this change? The reason we have separate entries for tx and rx is for interference. For capacity, they should have opposite sign.

Reply:
I think I agree with this change to have more accurate interference estimation. For capacity, I am a little bit confused now. For azimuth/horizontal scan, we take sector orientation into account. For elevation/vertical scan, do we have sector orientation? If all the devices are pointing to horizontal plane (as you mentioned below), then this is correct.

Reply:
We do not have elevation angle for sectors, just horizontal. Is this something we should correct?

Reply:
DN is always pointing toward horizontal plane (so no need to change). CNs are always pointing toward its DN. Not sure whether it is easy to include or not then.

Reply:
I'll see if I can make that change in a follow-up PR. Don't think it'll be too difficult.

I believe Wenteng at some point almost made that change but later undid it. I wonder why.

Reply:
Probably let's check with our partners first. They might change their deployment method from time to time when they actually put the devices in the field. But I agree with you: no need to do it in this PR. Thanks.